workshop_app/views.py

```diff
@@ -872,3 +877,15 @@ def testimonials(request):
 		messages = paginator.page(paginator.num_pages)
 	return render(request, 'workshop_app/testimonals.html', {"messages":messages})
 
+@login_required
+def scheduled_workshops(request):
+	user = request.user
+	if is_instructor(user) and is_email_checked(user):
+		accepted_workshops = ProposeWorkshopDate.objects.all().order_by('-id')[:15]
```

Comment:
I'll add a exception for that and [:15] is for giving only top 15 records after  reversing the list